src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if ((a <= 0 || b <= 0 || c <= 0)
+      || (a + b + c - Math.max(a, b, c)
```

Comment:
Better create a variable for this (Math.max(a, b, c))

Reply:
Ok, agree.

Reply:
Good point, please fix this. But even better would be to split this `if` to 2 different conditions with different Error messages(1 - when you got negative values, 2 - when sum of 2 sides are less than 3 side)